docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.9'
 services:
   tyk-dashboard:
-    image: tykio/tyk-dashboard:v5.3.0
+    image: tykio/tyk-dashboard:s5.4.0-alpha1
```

Comment:
@Keithwachira, @letzya has made a good suggestion of modifying this to use envar from .env

Reply:
will open a new pr